crates/cargo-util/src/registry.rs

```diff
@@ -42,4 +51,13 @@ mod tests {
         assert_eq!(make_dep_path("AbCd", false), "Ab/Cd/AbCd");
         assert_eq!(make_dep_path("aBcDe", false), "aB/cD/aBcDe");
     }
+
+    #[test]
+    fn test_10993() {
```

Comment:
Could you call out what this is testing either in place of or in addition to the issue?